runtime/src/bank.rs

```diff
@@ -4529,6 +4568,39 @@ impl Bank {
         }
     }
 
+    fn adjust_capitalization_for_existing_specially_retained_accounts(&self) {
+        use solana_sdk::{bpf_loader, bpf_loader_deprecated, secp256k1_program};
+        let mut existing_sysvar_account_count = 8;
+        let mut existing_native_program_account_count = 4;
+
+        if self.get_account(&sysvar::rewards::id()).is_some() {
+            existing_sysvar_account_count += 1;
```

Comment:
Heh, casual looking of coverage revealed I'm missing to test this branch... ;) b6aa4519ed42741561f3e61e4ff2bdc39ccc56e3

![image](https://user-images.githubusercontent.com/117807/101991296-085b8d00-3cef-11eb-90b9-5c8319354301.png)